image/user/db/settings_v2_002-devel.sql

```diff
@@ -21,7 +21,7 @@ INSERT OR IGNORE INTO settings_tab (path, value) VALUES
     ('gs_lock_screen_passcode_is_on', '1'),
     ('gs_display_language', 'English'),
     ('gs_input_language', 'English'),
-    ('gs_eula_accepted', '0'),
```

Comment:
Why set EULA accepted = true as default?
I see it is then set to false at the beginning of onboarding.

Reply:
@pawel-mudita is only set to true when compiling with the WITH_DEVELOPMENT_FEATURES flag because then onboarding is automatically skipped